interface/super/edit_list.php

```diff
@@ -1118,11 +1117,14 @@ function lister() {
                     }
                 </script>
                 <?php
-                    $urlFrom = ( isset($_GET["list_from"]) && intval($_GET["list_from"]) >= 0 ? intval($_GET["list_from"]) : (isset($_POST["list_from"]) && intval($_POST["list_from"]) >= 0 ? intval($_POST["list_from"]) : 0) );
-                    $urlTo   = ( isset($_GET["list_to"]) && intval($_GET["list_to"])  >= 0 ? intval($_GET["list_to"]) : (isset($_POST["list_to"]) && intval($_POST["list_to"]) >= 0 ? intval($_POST["list_to"]) : 10) );
+                    $urlFrom = ( isset($_REQUEST["list_from"]) && intval($_REQUEST["list_from"]) >= 0 ? intval($_REQUEST["list_from"]) :  1 );
+                    $urlTo   = ( isset($_REQUEST["list_to"]) && intval($_REQUEST["list_to"]) >= 0 ? intval($_REQUEST["list_to"]) : '' );
+                    if( $urlTo == 0){
+                        $urlTo = '';
+                    }
                 ?>
                 <div class="blck-filter" style="float: left; margin-top: 5px; margin-left: 10px; border:0px solid red; width: auto; ">
-                    <div id="input-type-from" style="float: left; "><?php echo xlt("From"); ?>&nbsp;<input autocomplete="off" id="list-from" value="<?php echo attr($urlFrom);?>" style = "margin-right: 10px; width: 40px;">
+                    <div id="input-type-from" style="float: left; "><?php echo xlt("From"); ?>&nbsp;<input aumtocomplete="off" id="list-from" value="<?php echo attr($urlFrom);?>" style = "margin-right: 10px; width: 40px;">
```

Comment:
above mispelled?

Reply:
Hi @bradymiller , This is a grammatically mistake